presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/avro/AvroRowEncoder.java

```diff
@@ -76,55 +77,55 @@ private boolean isSupportedType(Type type)
     @Override
     protected void appendNullValue()
     {
-        record.put(columnHandles.get(currentColumnIndex).getName(), null);
```

Comment:
I opted against falling back to `getName` if `getMapping` was missing to keep this simpler to implement and handle. But this is potentially a breaking change.